### ISSUE TITLE
FtpWebRequest is on Life Support Only

### ISSUE THREAD

**Issue by franhoey** [cake-contrib/Cake.Ftp#19]
According to the documentation FtpWebRequest is not recommended for new projects - https://docs.microsoft.com/en-us/dotnet/api/system.net.ftpwebrequest?view=netcore-2.2#remarks

Following the link there are a number of third party libraries recommended.

**Comment by franhoey** [comment]
I'm happy to pick this issue up as I could really use this library at the moment

**Comment by phillipsj** [comment]
Sure thing, feel free to replace it with what you think is the best choice.

**Comment by franhoey** [comment]
Looking through the Stackoverflow comments on the link from the MS docs page I'll use [FluentFTP](https://github.com/robinrodricks/FluentFTP). API looks good, docs are good, repo is recently maintained.

**Comment by phillipsj** [comment]
Sounds good to me.

**Comment by franhoey** [comment]
See pull request. I have replaced FtpWebRequest, and submitted it. There is a little more tidy up, but I thought you might want to have a look first.

- Replaced FtpWebRequest with FluentFtp
- Added Alias signatures as Uri no longer makes sense (I have kept the older aliases to ensure it's backwards compatible)
- Added to the FtpSettings to allow control of the connection type and some simple freebies like auto create directory

Still todo:

- Add FluentFtp to the nuspec (I just realised I forgot it)
- Add details to the Readme as settings are now a little more complex
